src/managers/db-manager.js

```diff
@@ -359,7 +360,7 @@ class DbManager {
 
     const permissionRows = await this.permissionsTable.getRows({
       source: BotTable.DISCORD_SOURCE,
-      subjectType: PermissionsManager.SUBJECT_TYPES.user.name,
+      subjectType: DiscordPermissionsManager.SUBJECT_TYPES.user.name,
```

Comment:
I think we can use base PermissionsManager class here - see my comment in slack-permission-manager module.